parsl/jobs/strategy.py

```diff
@@ -26,6 +26,10 @@ class ExecutorState(TypedDict):
     If the executor is not idle, then None.
     """
 
+    first: bool
+    """Is this the first poll for this executor?
+    """
+
```

Comment:
Stating what the boolean does without being explicit _can_ (though obviously not always) lead to ambiguous understanding.  Suggest being explicit, like:
```python
    first: bool  # True if executor has not yet been polled
```